src/v/config/node_overrides.cc

```diff
@@ -19,27 +19,35 @@
 namespace config {
 
 std::ostream& operator<<(std::ostream& os, const config::node_id_override& v) {
+    if (v.force) {
```

Comment:
on naming:

Wondering if "force" should be accompanied with a bit more context. It sorta gives an impression that UUID is force overridden .. which is not the case I think.. .. few names come to my mind.

- ignore_existing_node_id
- force_overwrite_existing_node_id
- overwrite_existing_node_id 

wdyt.

Reply:
yeah, agreed. `ignore_existing_node_id` seems fine

Reply:
done